plugin/adsk/plugin/importTranslator.cpp

```diff
@@ -158,6 +158,10 @@ const std::string& UsdMayaImportTranslator::GetDefaultOptions()
         std::vector<std::string> entries;
         for (const std::pair<std::string, VtValue> keyValue :
              UsdMayaJobImportArgs::GetDefaultDictionary()) {
+            if (keyValue.first == UsdMayaJobImportArgsTokens->shadingMode.GetString()) {
+                // Shading mode varies with loaded plug-ins. Remove from defaults.
+                continue;
+            }
```

Comment:
This is the value stored in Maya when the mayaUsdPlugin initializes. At this point in time, we have not yet finished loading all dependent USD plugins and have a very limited list of conversions to choose from.